### DISCUSSION TITLE
Does addin support export/import of custom ribbon?

### DISCUSSION THREAD

**Discussion by HughWarrington** [joyfullservice/msaccess-vcs-addin#544] [Q&A] [answered]
Title says it all really. I'm guessing the answer is no since I don't see anything in my full export.

Does anyone know what would be involved?

How popular are custom ribbons? I assumed they would be very popular in Access LOB apps as a way of navigating around the app. What do people use instead if anything?

**Comment by joyfullservice** [comment, marked as answer]
If you are wanting to export custom ribbons defined in the system table `USysRibbons`, you can simply include the table data (in XML format) in your export options. The ribbon should be restored when you build from source.

If you are referring to `CommandBar` menus, such as custom shortcut menus, there is a work in progress for supporting those. Access doesn't provide a simple native export for those, but `clsDbCommandBar` contains some of the groundwork for Command Bar export. Discussion #478 provides a little more context for those efforts.

**Reply by HughWarrington** [reply]
Thanks for the info. Having done a bit more research, I can see the project I have inherited is using the CommandBar approach. In my scenario it's possible to switch to the XML ribbon which will produce a better result in any case as well as being compatible with this addin.
